src/engine/Metavariable_pattern.mli

```diff
@@ -15,6 +15,7 @@ val get_nested_metavar_pattern_bindings :
   Range_with_metavars.t ->
   (* The arguments in CondNestedFormula *)
   Metavariable.mvar ->
+  (* Why is this xlang optional? *)
   Xlang.t option ->
   Rule.formula ->
   Metavariable.bindings list
```

Comment:
If you have an answer to the question above, I'll add a comment. It would be nicer if the `Xlang.t` argument wasn't optional or if there was a label hinting at its role.

Reply:
no idea ... maybe git blame can help.